### PR TITLE
Update README with livekit config

### PR DESCRIPTION
**Pull request by xmj**
In line with the livekit updates to GroupCallView etc, modify README.md to contain a working configuration sample.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,19 +90,19 @@ yarn dev
...
-```yaml
-"livekit": {
-    "server_url": "ws://localhost:7880",
-    "jwt_service_url": "http:/localhost:8881"
-  },
+```json
+,
+  "livekit": {
+      "livekit_service_url": "http://localhost:7881"
+    }
```
Comment (SimonBrandner) [inline README.md:105]:
```suggestion
,
{
  "livekit": {
    "livekit_service_url": "http://localhost:7881"
  }
}
```

**Review by SimonBrandner — APPROVED** [review]
Thank you!!

File: README.md (hunk 1 above)
Comment (xmj) [reply README.md:105]: This should probably be `http://localhost:7880" but I'm still having an issue with @dbkr's OpenID stuffs in the SFU code here:
https://github.com/vector-im/element-call/blob/livekit/src/livekit/openIDSFU.ts#L39

so I haven't been able to test that yet.

Which livekit-version do I need to get it running?

Comment (SimonBrandner) [reply README.md:105]: I think it was you with whom I chatted earlier? Are things clearer now?

**Comment by toger5** [comment]
@xmj are you happy in how the readme now looks?
I think this PR still has the change to the sample config which is nice.
Do you think you can update it to only have those changes?

**Comment by xmj** [comment]
> @xmj are you happy in how the readme now looks? I think this PR still has the change to the sample config which is nice. Do you think you can update it to only have those changes?

Hi

My usecase for Element-call went away over time - we did some hackathon @ hack4glarus.ch to get an instance running, that's where these changes came from.

happy for someone else to carry the torch :-D

**Comment by toger5** [comment]
Closed in favor of: https://github.com/element-hq/element-call/pull/2483

**Pull request closed without merging**